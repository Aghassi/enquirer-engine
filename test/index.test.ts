import index, * as allIndex from 'src';


describe("index exports", () => {
    it("default export should match snapshot", () => {
        expect(index).toMatchSnapshot()
    })

    it("all exports should match snapshot", () => {
        expect(allIndex).toMatchSnapshot()
    })
})