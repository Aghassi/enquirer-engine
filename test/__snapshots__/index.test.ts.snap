// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index exports all exports should match snapshot 1`] = `
Object {
  "default": "hello world",
}
`;

exports[`index exports default export should match snapshot 1`] = `"hello world"`;
